index.html

```diff
@@ -13,28 +13,25 @@
 
 <body>
     <div class="container">
+        <header>
+            <img src="img/lambda-black.png">
+        
         <nav class="nav">
             <div class="links">
                 <a class="nav-link" href="">Home</a>
-                <a class="nav-link" href="C:\Users\djzar\OneDrive\Desktop\Lambda School_WebPT5_projects\Sprint-Challenge--User-Interface-master\about.html" target="_blank" rel="noopener">About</a>
+                <a class="nav-link" href="about.html">About</a>
                 <a class="nav-link" href="">Products</a>
                 <a class="nav-link" href="">Blog</a>
                 <a class="nav-link" href="">Contact</a>
-                <a href=></a>
+                </a>
             </div>
         </nav>
-        
-    </div>
-
-        <div class="lambda-black">
-            <img src="C:\Users\djzar\OneDrive\Desktop\Lambda School_WebPT5_projects\Sprint-Challenge--User-Interface-master\img\lambda-black.png">
-        </div>    
+    </header>
 
         <div class="jumbo">
-            <img src="C:\Users\djzar\OneDrive\Desktop\Lambda School_WebPT5_projects\Sprint-Challenge--User-Interface-master\img\jumbo.jpg">
+            <img src="img\jumbo.jpg">
```

Comment:
this should be a forward slash "/" and not a backward slash "\". I know backward slashes are used in those windows paths...try to get in habit of using forward slashes in your code.